bigbone/build.gradle

```diff
@@ -1,16 +1,19 @@
 plugins {
     id 'bigbone.library-conventions'
     id 'bigbone.integrationtest-conventions'
+    alias(libs.plugins.kotlin.serialization)
 }
 
 dependencies {
     api libs.okhttp
-    api libs.gson
+    api libs.kotlinx.serialization.json
```

Comment:
Ah. Dang. I still had this as api here. 🙈

Reply:
Personally, I’d change this to `implementation` per my reasoning in the other comments. 

@andregasser Feel free to update this PR if you can. I won’t be near a computer until tomorrow likely but would like this to get merged soon.

Reply:
@andregasser I have changed this to `implementation` in ae8ab1c and have also removed the dependency on kotlinx serialization from both sample modules as they only used it for a log output. I’ve replaced that one with a simple `toString` which, as it is a `data class`, still provides helpful output just for testing the sample.